ksql-engine/src/main/java/io/confluent/ksql/schema/inference/DefaultSchemaInjector.java

```diff
@@ -72,7 +91,7 @@ public DefaultSchemaInjector(final TopicSchemaSupplier schemaSupplier) {
   }
 
   private SchemaAndId getValueSchema(
```

Comment:
why is this called "value schema"? as in kafka record value (as opposed to key)?

Reply:
no idea 🤷‍♂️ - honestly I didn't look at the innards of this class